knime-scripting-templates/R/snippet-templates.txt

```diff
@@ -592,21 +592,23 @@ rOut <- data.frame(do.call("cbind", new.l))
 </rgg>
 ########################################################################################
 # name: Mahalanobis Distance Calculator
+# author: Antje Janosch
 # category: distance measures
 
-Calculates mahalonobis distances to the mean of a control group (typically your negative controls). 
-Option: choose to use the inverted covariance matrix (default: auto) 
+Calculates the squared mahalonobis distances (MD) to the mean of a reference population (typically your negative controls). 
+Option 'Use inverted': If 'yes', the inverted covariance matrix will be used. If 'auto', the script will make a choice based on the determinant of the covariance matrix.
+Option 'Outlier p-value': The given p-value is used to calculate a MD threshold. Data rows with a MD above this threshold can be considered as outliers. 
 
 ATTENTION: 
-This snippet expects a column named 'treatment' with categorial values! Select a negative control name. This will be used if negative control column is set to -USE SELECTED TREATMENTS- (default).
+This snippet expects a column named 'treatment' with nominal domain values! If it's available, a reference population can be chosen. It will be used if 'Reference population column' is set to -USE SELECTED TREATMENTS- (default).
 
-Alternatively, the message 'Error: Could not find atrribute: treatment' can be ignored, a negative control column can be selected and the name of the negative control can be given as text.   
+Alternatively, the message 'Error: Could not find atrribute: treatment' can be ignored. A reference population column can be selected and the name of the reference population can be given as text. 
 ######
 <rgg>
 
 <!--1. Title and short description -->
 
-<h3 text="$$$TEMPLATE_NAME$$$" aligment="center" span="full"/>
+<h3 text="Mahalanobis Distance Calculator" aligment="center" span="full"/>
```

Comment:
is there a reason to use a fixed text rather then getting the name of the template?